cli/internal/cmd/upgradecheck.go

```diff
@@ -560,7 +559,7 @@ func (v *versionUpgrade) buildString() (string, error) {
 		fmt.Fprintln(&upgradeMsg, "")
 	}
 
-	if v.newServices != "" {
+	if v.newServices != (consemver.Semver{}) {
```

Comment:
```suggestion
	if v.newServices != consemver.Semver{} {
```
The parenthesis shouldn't be needed

Reply:
Couldn't make the compiler happy without them.